src/ios/CDVCapture.m

```diff
@@ -292,6 +295,65 @@ - (CDVPluginResult*)processVideo:(NSString*)moviePath forCallbackId:(NSString*)c
     return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:fileArray];
 }
 
+- (void)showAlertIfAccessProhibeted
+{
+    if (![self hasCameraAccess]) {
+        [self showPermissionsAlert];
+    }
+}
+
+- (BOOL)hasCameraAccess
+{
+    if (![AVCaptureDevice respondsToSelector:@selector(authorizationStatusForMediaType:)]) {
+        return YES;
+    }
```

Comment:
Just a note here regarding iOS support for this plugin. The code is sound, but this selector already exists since iOS 7. I'm wondering if we should add iOS engine restrictions to this plugin's plugin.xml instead of all this runtime checking.

FYI The latest cordova-ios only supports iOS 9 or greater.